db/migrate/20150629180552_change_password_type_user.rb

```diff
@@ -0,0 +1,6 @@
+class ChangePasswordTypeUser < ActiveRecord::Migration
+  def change
+    remove_column :users, :password
+    add_column :users, :password, :varchar, null: false
```

Comment:
How many migrations should exist to achieve this properly on a live system with minimal downtime?